modules/va_forms/app/workers/va_forms/form_builder.rb

```diff
@@ -0,0 +1,125 @@
+# frozen_string_literal: true
+
+require 'sidekiq'
+require 'va_forms/regex_helper'
+
+module VAForms
+  class FormBuilder
+    include Sidekiq::Worker
```

Comment:
Without [sidekiq_options](https://github.com/department-of-veterans-affairs/vets-api/blob/master/modules/appeals_api/app/workers/appeals_api/higher_level_review_upload_status_updater.rb#L11-L12) to limit the number of retries, it will default to around [21 days of retrying](https://github.com/sidekiq/sidekiq/wiki/Error-Handling#automatic-job-retry). I believe we said we only want to retry for ~1hr.